src/pgen/cluster.cpp

```diff
@@ -166,15 +227,13 @@ void ProblemGenerator(MeshBlock *pmb, parthenon::ParameterInput *pin) {
             ->Param<HydrostaticEquilibriumSphere<ClusterGravity, ACCEPTEntropyProfile>>(
                 "hydrostatic_equilibirum_sphere");
 
-    const auto P_rho_profile = he_sphere.generate_P_rho_profile<Kokkos::View<
-        parthenon::Real *, parthenon::LayoutWrapper, parthenon::HostMemSpace>>(ib, jb, kb,
-                                                                               coords);
+    const auto P_rho_profile = he_sphere.generate_P_rho_profile(ib, jb, kb, coords);
 
     // initialize conserved variables
-    for (int k = kb.s; k <= kb.e; k++) {
-      for (int j = jb.s; j <= jb.e; j++) {
-        for (int i = ib.s; i <= ib.e; i++) {
-
+    parthenon::par_for(
+        DEFAULT_LOOP_PATTERN, "cluster::ProblemGenerator::UniformGas",
```

Comment:
This is the else branch of the `UniformGas` conditional, isn't it?

Reply:
Yes, I can add a comment to clarify that.